batch/batch/server/globals.py

```diff
@@ -24,3 +26,19 @@ def next_id():
 
     _counter = _counter + 1
     return _counter
+
+
+_recent_events = []
+
+
+def get_recent_events():
+    return _recent_events
+
+
+def add_event(event):
+    event['time'] = str(datetime.datetime.now())
+
+    global _recent_events
+
+    _recent_events = _recent_events[:100]
```

Comment:
should be a deque: https://docs.python.org/3.3/library/collections.html#collections.deque